static/topbar.js

```diff
@@ -39,11 +39,12 @@ const mapStateToProps = (state) => {
             break;
         case BuildState.BUILT:
             visibleBrowseLink = true;
+            visibleHomeLink = false;
+            buildState = BuildState.BUILT;
```

Comment:
I don't think these changes make a difference - `visibleHomeLink` being `null` (from line 32) should be equivalent to it being `false` (I might be wrong though - I admit my understanding of JS truthiness semantics is not great). `buildState` should already be `BuildState.BUILT` from line 35 and the branch of the switch we're in.